tox.ini

```diff
@@ -41,13 +41,21 @@ commands =
   pytest --nbmake --nbmake-timeout=3000 {posargs} docs/
 
 [testenv:doctest]
-basepython = python3.10
```

Comment:
We actually run the doctests for various Python versions so we should not be hard-coding this.

Reply:
Related, I've wondered if we ought to use generative section names here so that a user could choose to run `py37-doctest`, just as the notebook environment is set up.